webnn/concat.https.any.js

```diff
@@ -0,0 +1,19 @@
+// META: title=test WebNN API concat operation
+// META: global=window,dedicatedworker
+// META: script=./resources/utils.js
+// META: timeout=long
+
+'use strict';
+
+// https://webmachinelearning.github.io/webnn/#api-mlgraphbuilder-concat
+
+const buildConcat = (builder, resources) => {
+  // MLOperand concat(sequence<MLOperand> inputs, long axis);
+  const inputOperands = [];
+  for (let input of resources.inputs) {
+    inputOperands.push(builder.input(input.name, {type: input.type, dimensions: input.shape}));
+  }
+  return {outputOperand: builder.concat(inputOperands, resources.axis)};
```

Comment:
Consider setting the output name from resources, e.g., `resources.expected.name`, instead of using a hardcoded `outputOperand` here. That would ensure the output name is consistent with `buildGraph()`.